sycl/test/scheduler/MultipleDevices.cpp

```diff
@@ -96,10 +98,9 @@ int main() {
   queue MyQueue2(HOSTSelector);
   int Result = -1;
   try {
-    cpu_selector CPUSelector;
-    MyQueue1 = queue(CPUSelector);
+    TestQueue MyQueue1 {default_selector()};
     Result = multidevice_test(MyQueue1, MyQueue2);
-    MyQueue2 = queue(CPUSelector);
+    TestQueue MyQueue2 {default_selector()};
```

Comment:
Isn't the `MyQueue2` going to have the same device as `MyQueue1`? Judging by the name of the test, it is supposed to run on different devices and ensure that memory and commands are scheduled correctly.
If so, then we need a selector that tries to select a different device for `MyQueue2`.

@romanovvlad, can you comment here?